mmv1/products/integrationconnectors/Connection.yaml

```diff
@@ -0,0 +1,1075 @@
+    message: "message"
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "location"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Location in which Connection needs to be created.
+  - !ruby/object:Api::Type::String
+    name: "name"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      Name of Connection needs to be created.
+
+properties:
+  - !ruby/object:Api::Type::String
+    name: "region"
+    description: |
+      The region of the Connection.
+    output: true
+  - !ruby/object:Api::Type::String
+    name: "description"
+    description: |
+      An arbitrary description for the Conection.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: "labels"
+    description: |
+      Resource labels to represent user provided metadata.
+  - !ruby/object:Api::Type::String
+    name: "connectorVersion"
+    required: true
+    description: |
+      connectorVersion of the Connector.
+  - !ruby/object:Api::Type::NestedObject
+    name: "status"
+    description: |
+      Status of the Integration Connector.
+    output: true
+    properties:
+      - !ruby/object:Api::Type::Enum
+        name: "state"
+        description: |
+          State of the Integration Connector
+        output: true
+        values:
+          - :STATE_UNSPECIFIED
+          - :CREATING
+          - :ACTIVE
+          - :INACTIVE
+          - :DELETING
+          - :UPDATING
+          - :ERROR
+          - :AUTHORIZATION_REQUIRED
+      - !ruby/object:Api::Type::String
+        name: "description"
+        description: |
+          Description of Status of Connection.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: "status"
+        description: |
+          Status of the Connection.
+        output: true
+
+  - !ruby/object:Api::Type::Array
+    name: "configVariable"
+    api_name: "configVariables"
+    description: |
+      Config Variables for the connection.
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: "key"
+          description: |
+            Key for the configVariable
+          required: true
+        - !ruby/object:Api::Type::Integer
+          name: "integerValue"
+          api_name: "intValue"
+          description: |
+            Integer Value of configVariable
+        - !ruby/object:Api::Type::String
+          name: "stringValue"
+          description: |
+            String Value of configVariabley
+        - !ruby/object:Api::Type::Boolean
+          name: "booleanValue"
+          api_name: "boolValue"
+          description: |
+            Boolean Value of configVariable
+        - !ruby/object:Api::Type::NestedObject
+          name: "secretValue"
+          description: |
+            Secret value of configVariable.
+          properties:
+            - !ruby/object:Api::Type::String
+              name: "secretVersion"
```

Comment:
can we mark this field required since this is the only subfield. We'd try to avoid allowing users send empty blocks.